hw1/hw1_script.py

```diff
@@ -0,0 +1,84 @@
+# This function checks whether input sequence is valid. If so, decides whether it's DNA or RNA
+def check(seq):
+    if 'u' in seq.lower() and 't' in seq.lower():
```

Comment:
Это довольно удобно можно сделать через множества, как и другие проверки на алфавит.
```suggestion
    if {'u', 't'} <= set(seq.lower()):
```
<= - оператор проверки на подмножество. В качестве альтернативы можно использовать метод `.issubset`